tableaudocumentapi/field.py

```diff
@@ -199,4 +200,9 @@ def _read_description(xmldata):
         if description is None:
             return None
 
-        return u'{}'.format(ET.tostring(description, encoding='utf-8'))  # This is necessary for py3 support
+        description_string = ET.tostring(description, encoding='utf-8')
+        # Format expects a unicode string so in Python 2 we have to do the explicit conversion
+        if sys.version_info[0] == 2:
+            description_string = description_string.decode('utf-8')
+
+        return u'{}'.format(description_string)  # This is necessary for py3 support
```

Comment:
I think I prefer a type check over a py version check

```
if isinstance(description_string, bytes):
    description_string = description_string.decode('utf-8')
return description_string
```

Then I believe the u'{}' format dance isn't necessary